flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnSessionCli.java

```diff
@@ -108,6 +111,11 @@
 
 	private final Options ALL_OPTIONS;
 
+	private static final String fileName = "yarn-app.ini";
+	private static final String cookieKey = "secureCookie";
+
+	private final Option SECURE_COOKIE_OPTION;
```

Comment:
This option can be retrieved from the main options. No need for yarn specific option. Please remove.